catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/cache/solr/impl/SortedQueryMonitor.java

```diff
@@ -128,10 +129,16 @@ public void run() {
           comparator = new DistanceResultComparator(sortOrder);
         } else if (Result.RELEVANCE.equals(sortType)) {
           comparator = new RelevanceResultComparator(sortOrder);
+        } else {
+          comparator =
+              Comparator.comparing(
+                  r -> getAttributeValue((Result) r, sortingProp.getPropertyName()),
+                  Comparator.<Comparable>nullsFirst(
```

Comment:
Do we want nulls first regardless of direction?  Maybe nulls last if DESCENDING?

Reply:
Will update code to do this.